src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyLineBreakAfter.java

```diff
@@ -26,10 +26,18 @@ else if (x < 0) {
                 else { break; }
                 switch (a)
                 {
-                case 0:
-                    break;
-                default:
-                    break;
+                    case 1:
+                    {
+                        break;
+                    }
+                    case (1 + 2):
+                    {
+                        break;
+                    }
+                    case 0:
+                        break;
+                    default:
```

Comment:
I think we also need to start validating `default` block in scope of this issue. Please add test cases for `default` too.

Reply:
@Vladlis This makes sense.
Since you are expanding the case of the issue, please update original issue with this too.
Release notes only go by issue title, so if we don't update it users won't know about us including `default` too.

Reply:
@rnveach done.
There is even a comment I didn't notice at first :)
https://github.com/checkstyle/checkstyle/issues/5207#issuecomment-345484643

Reply:
There is one more missing case: LITERAL_NEW.
```
        new Runnable() {
            @Override
            public void run() {/**/}
        };
```

Reply:
@pbludov , please move this out to separate issue, please recheck existing issues.